samples/samples.js

```diff
@@ -139,6 +139,9 @@
 		}, {
 			title: 'Toggle Scale Type',
 			path: 'scales/toggle-scale-type.html'
+		}, {
+			title: 'Axes Label',
```

Comment:
It should be "Axes Labels". (Either both words plural or both words singular. So either that or "Axis Label". But we use "Labels" above, so the plural form seems more consistent)